arwen/elrondapi/elrondei.go

```diff
@@ -377,9 +383,56 @@ func transferValue(context unsafe.Pointer, gasLimit int64, destOffset int32, val
 	metering.UseGas(gasToUse)
 
 	invBytes := arwen.InverseBytes(value)
-	invValue := big.NewInt(0).SetBytes(invBytes)
-	boundGasLimit := metering.BoundGasLimit(gasLimit)
-	output.Transfer(dest, send, boundGasLimit, invValue, data)
+
+	output.Transfer(dest, send, 0, big.NewInt(0).SetBytes(invBytes), data)
+
+	return 0
+}
+
+//export asyncCall
+func asyncCall(context unsafe.Pointer, destOffset int32, valueOffset int32, dataOffset int32, length int32) int32 {
```

Comment:
no need for return as this is a blocking function and it will get out from the wasmer context.

Reply:
Changed.